ItemDescription3/tableEXT/salesInvoiceLineTabExt.50402.TabExt.al

```diff
@@ -0,0 +1,28 @@
+tableextension 50402 "BCP salesInvoiceLineTabExt" extends "Sales Invoice Line"
+{
+    fields
+    {
+
+        modify("No.")
+        {
+            trigger OnAfterValidate()
```

Comment:
This code is not required. This code doing nothing coz Sales Invoice Line is not editable table.